landscape/how-to/install-landscape-server-on-microsoft-azure.md

```diff
@@ -0,0 +1,202 @@
+"publicIPAllocationMethod": "Static",
+"resourceGroup": "Landscape-rg",
+```
+
+Copy the IP address and set it as the A record value for the domain or subdomain that will serve as the FQDN. You set the A record in your DNS service. You can also use Azure DNS zone to host your DNS domain and manage your DNS records.
+
+Verify the A record using `nslookup`. Replace `{landscape.domain.com}` with your FQDN:
+
+```
+nslookup {landscape.domain.com}
+```
+
+You’ll receive output similar to:
+```
+Server:		127.0.0.53
+Address:	127.0.0.53#53
+   
+Non-authoritative answer:
+Name:	landscape.domain.com
+Address: 34.XXX.XXX.XXX
+```
+
+If the address value in the `nslookup` output matches the value of the `LandscapePublicIP` static IP address, the LetsEncrypt SSL provisioning step defined in the cloud-init configuration automation template will succeed.
+
+
+## Deploy Landscape Server VM with cloud-init
+
+Before beginning the deployment process with cloud-init, you must choose which of the two cloud-init configuration automation templates you want to use. In the [Landscape Scripts](https://github.com/canonical/landscape-scripts) Github repository, there are two Landscape Quickstart cloud-init configuration templates: [`cloud-init-quickstart.yaml`](https://github.com/canonical/landscape-scripts/blob/main/provisioning/cloud-init-quickstart.yaml) and [`cloud-init-quickstart-fips.yaml`](https://github.com/canonical/landscape-scripts/blob/main/provisioning/cloud-init-quickstart-fips.yaml).
+
+The `cloud-init-quickstart.yaml` template is designed for anyone, and the `cloud-init-quickstart-fips.yaml` is designed for FIPS compliant deployments of Landscape Server. For more information, see [how to install FIPS-compliant Landscape Server](https://ubuntu.com/landscape/docs/install-fips-hardened-landscape-server).
+
+Once you’ve chosen your configuration template, complete the following steps.
+
+1. Set the `IMAGE_FAMILY` environment variable based on the cloud-init configuration you chose.
+
+- If you’re using `cloud-init-quickstart.yaml`, run:
+```
+curl -s https://raw.githubusercontent.com/canonical/landscape-scripts/main/provisioning/cloud-init-quickstart.yaml -o cloud-init.yaml
+IMAGE_FAMILY=Canonical:0001-com-ubuntu-pro-jammy:pro-22_04-lts-gen2:latest
+```
+
+ - If you’re using `cloud-init-quickstart-fips.yaml`, run:
+```
+curl -s https://raw.githubusercontent.com/canonical/landscape-scripts/main/provisioning/cloud-init-quickstart-fips.yaml -o cloud-init.yaml
+IMAGE_FAMILY=Canonical:0001-com-ubuntu-pro-focal-fips:pro-fips-20_04-gen2:latest
+```
+
+2. Open the downloaded cloud-init YAML file in an editor, determine which configuration parameters need to be changed between lines 4 and 32 and change these parameters.
+
+The `HOSTNAME` on line 16 and `DOMAIN` on line 19 must be changed. Updating `EMAIL` on line 9, and adding your SendGrid API key on line 29 as the `SMTP_PASSWORD` are optional, but strongly recommended.
+
+
+### Create VM
+
```

Comment:
```suggestion

Note: Microsoft recommends not storing sensitive data in the `--custom-data` field. For more information, see [Microsoft's documentation on custom data and cloud-init for virtual machines](https://learn.microsoft.com/en-us/azure/virtual-machines/custom-data).

```

Reply:
If you have any additional thoughts on how this information could be noted to the user, feel free to provide other suggestions. :) I just want to make sure the user isn't accidentally exposing sensitive data without realizing.